### PR TITLE
ISSUE_TEMPLATE/testing.md: more tweaks

### PR DESCRIPTION
**Pull request by jlebon**
Some tweaks from lessons learned from a new release.

---

Don't merge yet, will likely have other tweaks as I go through the release.

### CODE REVIEW

**Comment by dustymabe** [comment]
Let's update `Run kola on AMI to sanity check` to something like `Check Kola AWS run to make sure it didn't fail`.

**Comment by jlebon** [comment]
OK, lifted hold on this now!

File: ISSUE_TEMPLATE/testing.md [hunk 1]
```diff
@@ -60,19 +59,27 @@ upgrade` will have the new update.
...
 
 ```
 aws s3 ls --recursive s3://fcos-builds/prod/streams/testing/builds/$VERSION/
 curl -I https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/$VERSION/x86_64/fedora-coreos-$VERSION-qemu.qcow2.xz.sig
+```
+
+  If the ACL for the signatures are wrong, here's a gnarly one-liner to fix them all:
+
+```
+aws s3api list-objects --bucket fcos-builds --prefix prod/streams/testing/builds/$VERSION --output json | jq '.Contents[].Key' -r | grep '.sig$' | xargs -n1 aws s3api put-object-acl --bucket fcos-builds --acl public-read --key
```
Comment (dustymabe) [inline ISSUE_TEMPLATE/testing.md]: I wish we knew what was causing this. Do you think the permissions mohan has aren't enough?

File: ISSUE_TEMPLATE/testing.md [hunk 2]
```diff
@@ -98,13 +105,12 @@ fedora-coreos-stream-generator -releases=https://fcos-builds.s3.amazonaws.com/pr
...
 - [ ] Commit the changes and open a PR against the repo.
+- [ ] Post a link to the PR as a comment to this issue
 - [ ] Wait for the PR to be approved.
 - [ ] Once approved, merge it and push the content to S3:
 
 ```
-aws s3 cp --acl public-read --cache-control 'max-age=60' streams/testing.json s3://fcos-builds/streams/testing.json
-
-aws s3 cp --acl public-read --cache-control 'max-age=60' updates/testing.json s3://fcos-builds/updates/testing.json
+aws s3 sync --acl public-read --cache-control 'max-age=60' --exclude '*' --include 'streams/*' --include 'updates/*' . s3://fcos-builds
```
Comment (dustymabe) [inline ISSUE_TEMPLATE/testing.md]: This felt safer before when we were targeting a single file. What if someone has some junk files in their repo? I think I'd feel better about leaving ti the way it was.

Comment (dustymabe) [reply ISSUE_TEMPLATE/testing.md]: That is unless there is some other improvements here other than collapsing two commands into one.

Comment (jlebon) [reply ISSUE_TEMPLATE/testing.md]: I guess so. I mean, it's two commands right now, but eventually it'd be 6 commands when we add the other two streams. Seemed cleaner to just provide the one-liner.

But, I'm really hoping we get to a place soon where we don't run this command by hand and just have a syncer take care of it. So, meh... I can switch it back if you'd like.

File: ISSUE_TEMPLATE/testing.md (hunk 1 above)
Comment (jlebon) [reply ISSUE_TEMPLATE/testing.md]: I'm not sure, but thankfully we'll (hopefully) only have to go through this once or twice more.

**Pull request merged**